moveit_ros/planning_interface/test/CMakeLists.txt

```diff
@@ -14,19 +14,7 @@ if (CATKIN_ENABLE_TESTING)
   target_link_libraries(subframes_test moveit_move_group_interface ${catkin_LIBRARIES})
 
   add_rostest(python_move_group.test)
-  add_rostest(python_move_group_ns.test)
```

Comment:
Just to be sure: This test file still exists, so why don't we execute it?

Reply:
Well spotted! This should be added again. (It was removed in between because I tried to handle both tests with the same .test file.)